src/Microsoft.TestPlatform.CoreUtilities/Output/OutputExtensions.cs

```diff
@@ -116,6 +116,11 @@ private static void SetColorForAction(ConsoleColor foregroundColor, Action actio
             return;
         }
 
+#if NET5_0_OR_GREATER
+        if (OperatingSystem.IsAndroid() || OperatingSystem.IsIOS() || OperatingSystem.IsBrowser() || OperatingSystem.IsTvOS())
+            return; // Console color not supported on these platforms.
```

Comment:
```suggestion
            // Console color not supported on these platforms.
            action.Invoke();
```

Reply:
It is probably better to still write what we wanted to write and just avoid coloring it.

Reply:
Oh yeah of course. Doh! (Albeit id prefer class libraries didn't write to a console)